src/lib/storage/fixed_string_dictionary_segment/fixed_string_vector.cpp

```diff
@@ -11,6 +11,12 @@
 
 namespace opossum {
 
+FixedStringVector::FixedStringVector(const FixedStringVector& other, const PolymorphicAllocator<char>& allocator)
+    : _string_length(other._string_length), _chars(other._chars, allocator), _size(other._size) {
+  // We need to set `_chars` in the initializer list. Otherwise, it would be first created with the default allocator
+  // and the assignment operator would not exchange the allocator.
```

Comment:
```suggestion
```

Reply:
Not sure what your suggestion is. The point is that `operator=` does not modify the allocator. Maybe that should be added?

Reply:
Ah, what Github is slightly confusing. Just saw the comment on the other line. Maybe:

"For pmr_vectors, `operator=` does not change the allocator. As such, we need to set `_chars` in the initializer list. Otherwise, it would be first created with the default allocator and the allocator that is passed in would be ignored."?

Reply:
Maybe:
"For pmr_vectors, operator= does not change the allocator. As such, we need to set _chars in the initializer list. Otherwise, it would be created using the default allocator and ignoring the passed in allocator."